setup.json

```diff
@@ -18,12 +18,16 @@
             "ase.ase = aiida_ase.calculations.ase:AseCalculation"
         ],
         "aiida.parsers": [
-            "ase.ase = aiida_ase.parsers.ase:AseParser"
+            "ase.ase = aiida_ase.parsers.ase:AseParser",
+            "ase.gpaw = aiida_ase.parsers.ase:GPAWParser"
+        ],
+        "aiida.workflows": [
+            "ase.gpaw.base = aiida_ase.workflows.base:BaseGPAWWorkChain"
         ]
     },
     "python_requires": ">=3.7",
     "install_requires": [
-        "aiida-core[atomic_tools]~=1.0"
+        "aiida-core[atomic_tools]==1.6.4"
```

Comment:
Wouldn't pin this to a patch version. Any reason you did this? Would probably at least just make this `~=1.6`

Reply:
A good question! I am not entirely sure why I chose `1.6.4`, maybe it was because I hadn't tested it on anything below - but - as you mention - I do not immediately see why it should be fixed at this specific patch. Also seems like some of the tests are failing with this change, will take a look, thanks!